src/dispatcher/dispatcher.js

```diff
@@ -43,6 +45,10 @@ class Dispatcher {
             console.log("inside mysql")
```

Comment:
Remove the unnecessarily added console.log

Reply:
we require this code